### PR TITLE
Change data schema

### PR DESCRIPTION
**Pull request by jiayu-joyce-he**

### CODE REVIEW

File: app/javascript/reactcomponents/Feedback.js [hunk 1]
```diff
@@ -1,14 +1,36 @@
...
 
 const Feedback = (props) => {
-  const current_sun_exposure = 2
-  const max_sun_exposure = 10
-  const skintone=6
+  const current_input = props.data[props.data.length-1]
+  const {safe_exposure_time, uv, uv_max, hours_in_sun, sun_block_application} = current_input
+  const message = sun_block_application ? "Great job applying sunblock and make sure to re-apply if you plan to stay in the sun!" : "Protect your skin and remember to apply sunblock!"
+
+  console.log(current_input.safe_exposure_time);
```
Comment (Benjamin-G) [inline app/javascript/reactcomponents/Feedback.js]: Delete Console

File: app/javascript/pages/Dashboard.js [hunk 2]
```diff
@@ -18,25 +18,32 @@ class Dashboard extends React.Component {
        return resp.json()
      })
      .then((data) => {
-       this.setState({ combinedData: data })
-       console.log("this is data", data);
-       console.log("this is state", this.state.combinedData);
+       this.setState({ data: data })
+       console.log("this is state", this.state.data);
```
Comment (Benjamin-G) [inline app/javascript/pages/Dashboard.js]: Delete Console

File: app/javascript/reactcomponents/Emoji.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ import React from "react"
 
 const Emoji = (props) => {
   const severity = Math.floor(((props.max_sun_exposure - props.current_sun_exposure)/props.max_sun_exposure) * 4)
-  console.log(severity);
+  // console.log(severity);
```
Comment (Benjamin-G) [inline app/javascript/reactcomponents/Emoji.js]: Delete Console

File: app/javascript/pages/Dashboard.js [hunk 4]
```diff
@@ -7,7 +7,7 @@ class Dashboard extends React.Component {
   constructor(props){
        super(props)
        this.state = {
-         combinedData: [],
+         data: null,
```
Comment (winescout) [inline app/javascript/pages/Dashboard.js]: What is 'data'?  Can we use a more descriptive Variable name here?

Comment (Benjamin-G) [reply app/javascript/pages/Dashboard.js]: This will be resolved this afternoon! 🥇

**Pull request merged**